pom.xml

```diff
@@ -279,6 +279,12 @@
         <artifactId>gson</artifactId>
         <version>2.8.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
@jeremybmerrill I fixed the issue with testBatchExtractor. Looks like in the changes I forgot we had formatting in place for batch and now to keep in line with rest of Tabula, BatchWriter follows CSVWriter which messed up the expected output in testBatchExtractor.